cppa_slack_transcript_tracker/utils/slack_tokens.py

```
@@ -0,0 +1,262 @@
+    """Check if Docker Selenium container is accessible."""
+    import socket
+    import urllib.error
+    import urllib.request
+
+    try:
+        raw_url = getattr(settings, "SELENIUM_HUB_URL", "http://localhost:4444/wd/hub")
+        selenium_hub_url = _validate_selenium_hub_url(raw_url)
+        base_url = selenium_hub_url.replace("/wd/hub", "")
+        status_url = f"{base_url}/status"
+        try:
+            response = urllib.request.urlopen(status_url, timeout=10)
```

Ruff (0.15.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)